packages/mobile/src/tokens/reducer.ts

```diff
@@ -0,0 +1,45 @@
+import { RehydrateAction } from 'redux-persist'
+import { getRehydratePayload, REHYDRATE } from 'src/redux/persist-helper'
+import { Actions, ActionTypes } from 'src/tokens/actions'
+
+export interface Token {
```

Comment:
I didn't do it yet, but this model will contain also the current usd price in a number field. We can call it `usdPrice`. And it should be optional (because we might not have enough information to obtain a accurate price)

Reply:
ok, added :)